server.js

```diff
@@ -67,7 +67,7 @@ app.get('/project/featured', getFeatured);
 app.get('/project/my', protect, getMyProjects);
 app.get('/project/:id', getProject);
 app.get('/project', getProjectList);
-app.delete('/project/:id', protect, protectProject('author'), deleteProject);
+app.delete('/project/:id', protect, deleteProject);
```

Comment:
This allows anyone to delete a project.  You need to modify the protectProject middleware